oauth2_http/java/com/google/auth/oauth2/MtlsConfig.java

```diff
@@ -0,0 +1,58 @@
+package com.google.auth.oauth2;
+
+import com.google.errorprone.annotations.CanIgnoreReturnValue;
+
+/** Holds an mTLS configuration (consists of address of S2A) retrieved from the Metadata Server. */
+public final class MtlsConfig {
```

Comment:
I think this should be re-branded to `S2AConfig`. `MtlsConfig` is much more generic than the settings in this class.

Reply:
Done in https://github.com/googleapis/google-auth-library-java/pull/1400/commits/359fd432480bfd92440ee680e1459bfe0e782fb9